src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -1,49 +1,97 @@
 import { Component, OnDestroy } from '@angular/core';
-import { CurrentContentService } from 'src/app/shared/services/current-content.service';
+import { CurrentContentService, EditAction } from 'src/app/shared/services/current-content.service';
 import { ItemData, ItemDataSource } from '../../services/item-datasource.service';
 import { FormBuilder, Validators } from '@angular/forms';
 import { Subscription } from 'rxjs';
 import { filter } from 'rxjs/operators';
 import { FetchError, Fetching, isReady, Ready } from '../../../../shared/helpers/state';
+import { UpdateItemStringService } from '../../http-services/update-item-string.service';
+import { Item } from '../../http-services/get-item-by-id.service';
+import { TOAST_LENGTH } from '../../../../shared/constants/global';
+import { MessageService } from 'primeng/api';
+import { ERROR_MESSAGE } from '../../../../shared/constants/api';
 
 
 @Component({
   selector: 'alg-item-edit',
   templateUrl: './item-edit.component.html',
-  styleUrls: ['./item-edit.component.scss']
+  styleUrls: ['./item-edit.component.scss'],
+  providers: [MessageService]
```

Comment:
They are working now but in the case of success, the page is changed too fast to actually see it. This could be an easy fix if I have to add a new EditAction.

Reply:
Not fixed?

Reply:
I haven't been able to make it work when the component is changed, but if an error happens, the notification will corectly show up

Reply:
But so it does not work well as it is not displayed correctly on success. Do you want me to check?

Reply:
Yes please